frontend/components/GamesVisibilityList.tsx

```diff
@@ -2,50 +2,40 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 import { Dispatch, AnyAction } from 'redux';
 import { AppStore, GameVisibilityActions } from '../storage/common';
-import { Game } from '../storage/models/Game';
+import { CheckboxRowProps, CheckboxRow } from './generics/CheckboxRow';
 interface GamesVisibilityListProps {
-	gameVisibilityToggles: Array<GameVisibilityProps>;
+	checkboxRows: Array<CheckboxRowProps>;
 	dispatch: Dispatch<AnyAction>;
 }
 const GamesVisibilityList: React.FC<GamesVisibilityListProps> = (props: GamesVisibilityListProps) => {
-	const { gameVisibilityToggles } = props;
+	const callbackFunction = (name: string, checkValue: boolean) => {
```

Comment:
Note that this function is recreated each time this component is rendered. It's not the end of the world but that's something we'll fix in the future, for now leave it as-is.